bentoml/_internal/utils/ngrok.py

```diff
@@ -82,16 +81,21 @@ def _download_ngrok(ngrok_path):
         url = "https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip"
     else:
         raise Exception(f"{system} is not supported")
-    download_path = _download_file(url)
+    download_path = download_file(url)
     with zipfile.ZipFile(download_path, "r") as zip_ref:
         zip_ref.extractall(ngrok_path)
 
 
-def _download_file(url):
+def download_file(url: str) -> str:
```

Comment:
why renaming these functions? I think they are clearly only internal to this module and not meant to be used by other BentoML modules, l'd suggest we keep the leading `_` here

Reply:
will do